Auth0/Management.swift

```diff
@@ -26,29 +26,25 @@ struct Management: Trackable, Loggable {
 
     func managementObject(response: Response<ManagementError>, callback: Request<ManagementObject, ManagementError>.Callback) {
         do {
-            if let dictionary = try response.result() as? ManagementObject {
+            if let dictionary = try response.result()?.body as? ManagementObject {
                 callback(.success(dictionary))
             } else {
                 callback(.failure(ManagementError(from: response)))
             }
-        } catch let error as ManagementError {
```

Comment:
Since the internal `Response` type now makes use of Swift 6's typed throws, the handlers no longer need that last catch: `callback(.failure(ManagementError(cause: error)))`.